### PR TITLE
Embed URL

### PR DESCRIPTION
**Pull request by gidsg**
Add Embed URL `<textarea>` to published atoms. This is useful as the link URL fails in Composer because it includes an API key.

CC @Reettaphant @akash1810 
screen shot 2016-09-26 at 10 27 26

### CODE REVIEW

File: public/javascript/atom/atom.html [hunk 1]
```diff
@@ -11,16 +11,19 @@
         <div class="col-md-6 asset-data">
             <h1 class="atom-section" id="section">Atom Data</h1>
 
-            <label ng-if="!publishedAtom" for="embedLink">Link to capi:</label>
-            <label ng-if="publishedAtom" for="embedLink">Embed link:</label>
+            <label for="embedLink">Link to capi:</label>
             <div id="embed-link">
-                <a
+            <a
                     href="{{linkToCapi}}"
```
Comment (akash1810) [inline public/javascript/atom/atom.html]: Though its not part of this PR, this should be [`ng-href`](https://docs.angularjs.org/api/ng/directive/ngHref).

**Comment by akash1810** [comment]
@gidsg you happy for this to be merged?

**Comment by gidsg** [comment]
Yes, sorry for delay @Reettaphant
